packages/analysis-report/src/ContentAnalysis.js

```diff
@@ -94,7 +95,7 @@ class ContentAnalysis extends React.Component {
 					this.renderCollapsible( __( "Errors", "wordpress-seo" ), headingLevel, errorsResults )
 				}
 				{ problemsFound > 0 &&
-					this.renderCollapsible( __( "Problems", "wordpress-seo" ), headingLevel, problemsResults )
+					this.renderCollapsible( __( "Problems", "wordpress-seo" ), headingLevel, [ ...upsellResults, ...problemsResults ] )
```

Comment:
With this check, when there is no problem found, the upsell bullet for Keyphrase distribution won't appear. 
<img width="603" alt="Screenshot 2022-06-16 at 11 17 45" src="https://user-images.githubusercontent.com/48715883/174037429-edeb1748-83a1-4509-826c-6e557a7d8b82.png">

Reply:
Good catch!